tikv-client/scripts/proto.sh

```diff
@@ -15,9 +15,10 @@
 #
 
 if [ -d "kvproto" ]; then
-    cd kvproto; git pull origin master; cd ..
-else 
-    git clone https://github.com/pingcap/kvproto
+    # cd kvproto; git pull origin master; cd ..
+    cd kvproto; git checkout 497dd34f807bccdb5bd01009c62c5fd5ced404f3; cd ..
+else
+    git clone https://github.com/pingcap/kvproto; cd kvproto; git checkout 497dd34f807bccdb5bd01009c62c5fd5ced404f3; cd ..
```

Comment:
I purpose that we use a stable version of kvproto rather than pulling the master build every time.

Reply:
It's fine for now, but keep yourself in the loop when KV side change proto def.

Reply:
@ilovesoup Yes I will. I would like to consider reverting this change back after I have finished Java test for kvproto. But not for now.